aws/rhel-8.10-ga-x86_64/.#config.json

```diff
@@ -0,0 +1 @@
+sanne@worky.4070:1716798465
```

Comment:
Probably should not be part of the PR...

Reply:
woops ._.